Header header

bool acc_switch_status
bool aeb_switch_status
bool lc_switch_status

float64 target_accel
bool acc_valid
bool hold_target_speed
float64 speed_setPoint

bool aeb_valid
bool aeb_override

bool lc_valid
float64 wheel_angle

uint64 alive_rolling_counter
